README.md

```diff
@@ -97,7 +97,13 @@ Upgrade `synctl` to a specified version.
 pip3 install --upgrade synctl==<version>
 ```
 
+# Size the PoP hardware configuration
+synctl can be used to find the size of the PoP hardware configuration.
 
+### synctl pop-sizing Syntax
+```
+synctl --pop-sizing-estimate
+```
```

Comment:
Hi @rui1234 @swethalohith2 I recommend we use a sub command not a single option here, for example `synctl get cost` or use a separate command for cost estimation like `synctl cost estimate --options`, to keep same with other command, and it is easy to extend more options with command.